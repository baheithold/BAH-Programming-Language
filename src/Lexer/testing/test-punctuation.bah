# This file tests valid punctuation


(
)
{
}
,
:
;
