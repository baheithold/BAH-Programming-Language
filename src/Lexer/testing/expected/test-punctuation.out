OPAREN
CPAREN
OBRACE
CBRACE
COMMA
COLON
SEMICOLON
END_OF_INPUT
